client/my-sites/stats/feedback/index.tsx

```diff
@@ -248,4 +244,43 @@ function StatsFeedbackController( { siteId }: FeedbackProps ) {
 	);
 }
 
-export default StatsFeedbackController;
+function logFeedbackPresentationStatus( key: string, value: boolean ) {
+	const isCalypsoLive = window.location.host === 'calypso.live';
+	const isCalypsoLocalhost = window.location.host.startsWith( 'calypso.localhost' );
+	if ( isCalypsoLive || isCalypsoLocalhost ) {
+		localStorage.setItem( key, value ? 'true' : 'false' );
+	}
+}
+
+const FEEDBACK_HIGH_TRAFFIC_SITE_THRESHOLD = 10000;
+
+function getHighTrafficThreshold() {
+	const value = Number( localStorage.getItem( 'StatsHighTrafficThreshold' ) );
+	return value !== 0 ? value : FEEDBACK_HIGH_TRAFFIC_SITE_THRESHOLD;
+}
+
+interface FeedbackPresentorProps {
+	siteId: number;
+}
+
+function StatsFeedbackPresentor( { siteId }: FeedbackPresentorProps ) {
+	const { supportCommercialUse } = useStatsPurchases( siteId );
+	const { data, isSuccess } = useFetchTrafficHook( siteId );
+
+	const visitors = data?.past_thirty_days.visitors ?? 0;
+	const highTrafficThreshold = useMemo( () => getHighTrafficThreshold(), [] );
+
+	const isHighTrafficSite = isSuccess && visitors > highTrafficThreshold;
+	logFeedbackPresentationStatus( 'StatsHighTrafficSite', isHighTrafficSite );
+
+	const isOdysseyStats = config.isEnabled( 'is_running_in_jetpack_site' );
+	const presentForHighTrafficSite = isOdysseyStats && isHighTrafficSite;
```

Comment:
I believe we should show the panel for Jetpack sites not Odyssey Stats here, as Dotcom sites run Odyssey as well.

Reply:
I'm guessing we need a combination of `isJetpack` and `! isAtomic` but I'll try and run this through the full testing matrix this afternoon.

Reply:
After testing across all the scenarios I could think of, I'm not seeing a case where the above `isOdysseyStats` value is `true` outside of self-hosted wp-admin. ie: It looks like the correct environment test to me.

I tested the following:

- self-hosted via wp-admin
- self-hosted via Calypso
- dotcom with Business plan (no plugins)
- dotcom with Business plan (yes plugins/atomic)
- P2 and…
-  jetpack.com.

If I've missed an environment I should test against, please let me know.

It's also worth noting that we limit the Jetpack upsells, which should only be shown inside wp-admin, with a check against `isOdysseyStats` too. — [link](https://github.com/Automattic/wp-calypso/blob/5c08326bc5ec015d201eb4468ee18b78520d0f43/client/my-sites/stats/site.jsx#L316)

In that example, I think the `&& ! isAtomic` might be redundant as my test Atomic site returns isAtomic=true, isJetpack=true, and isOdysseyStats=false.

Reply:
We  should show the feedback panel by by site types instead of environments. Simple sites and WoA sites have WP Admin as well where it runs Odyssey, but we don’t want to show it to them yet. We show the panel for all Jetpack sites regardless of whether they are in WPA or Calypso. Do that make sense?

Reply:
Ah, okay. That makes sense. I'll get it updated to evaluate based on site type.